docs/accuracy_aware_model_training/EarlyExitTrainig.md

```diff
@@ -0,0 +1,35 @@
+# EarlyExit training loop in NNCF
+
+Early-Exit training loop aims to get the compressed model with a desired accuracy criteria as earlier as possible. After NNCF initialization algorithms applied and then after each trainig epoch the loop measures accuracy of the compressed model and if it is reach the user-defined criteria exits and returns a compressed model.
```

Comment:
New version of this paragraph `Early-Exit training loop aims to get the compressed model with the desired accuracy criteria as earlier as possible. The idea is to check accuracy after each training epoch and exit the training process once the accuracy reaches the user-defined criteria. This pipeline is simple but effective. For many models, it reduces training time till just an initialization step of the algorithm.`

Reply:
I don't stand for the phrase: `The idea is to check accuracy after each training epoch and exit the training process once the accuracy reaches the user-defined criteria`
I suggest this version of the paragraph:
`Early Exit training loop aims to get the compressed model with the desired accuracy criteria as earlier as possible. This is done by checking a compressed model accuracy after each training epoch step and also after the initialization step then exits the fine-tuning process once the accuracy reaches the user-defined criteria`